docs/src/snap/howto/security/hardening.md

```diff
@@ -0,0 +1,38 @@
+# {{product}} hardening guide
+
+The {{product}} hardening guide provides actionable steps to enhance the
+security posture of your deployment. These steps are designed to help you align
+with industry-standard frameworks such as CIS and DISA STIG.
+
+{{product}} aligns with many security recommendations by
+default. However, since implementing all security recommendations
+would comes at the expense of compatibility and/or performance we expect
+cluster administrators to follow post deployment hardening steps based on their
+needs.
+
+This how-to has both the recommended minimum hardening steps and also a more
+comprehensive list of manual tests.
+
+Please evaluate the implications of each configuration before applying it.
+
+## Post-deployment hardening steps
+
+These steps are common to the hardening process for both CIS and DISA STIG
+compliance.
+
+```{include} ../../../_parts/common_hardening.md
+```
+
+
+## CIS and DISA STIG Hardening 
+
+If you would like to apply further DISA STIG recommendations please see [additional DISA STIG-specific steps].
+
+To asses compliance to the CIS hardening guidelines please see the [CIS assessment page].
+
+<!-- Links -->
+[Post-Deployment Configuration Steps section]:#post-deployment-configuration-steps
```

Comment:
I don't see this referenced anywhere?

Even if it were, shouldn't it have been `#post-deployment-harnening-steps`?

Reply:
Removed